charts/castai-agent/values.yaml

```diff
@@ -168,3 +168,13 @@ containerSecurityContext:
   capabilities:
     drop:
       - ALL
+
+topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        app.kubernetes.io/name: castai-agent
```

Comment:
I think you should use `{{ include "castai-agent.selectorLabels" . }}` instead of the hardcoded label to support changing labels via props

Reply:
Resolving because I didn't notice that this is in values.yaml. I think it's possible to still use variables in values.yaml but you need a tpl func which isn't worth it IMO.